docs/dev-guide/getting-started.md

```diff
@@ -1,11 +1,49 @@
 # Developer Guide - Getting Started
 
-Getting started text. Also, add additional pages as needed.
+* Prereqs
+    * Java 8
+    * Maven 3.x
+    * Git Tools
+    * Eclipse _Optional_
+
+## Prereqs
+
+### Java 8 (or above) 
+
+The quality measure and cohort engine has been tested with OpenJDK8 and OpenJDK11. To install Java on your system, we recommend downloading and installing Java 11 OpenJ9 from [https://adoptopenjdk.net](https://adoptopenjdk.net)
+
+### Maven 3.x (or above)
+
+The quality measure and cohort engine is a multi-module Maven project. Any recent version of Maven should work. The team uses 3.6.3. Download from the [Maven download site](https://maven.apache.org/download.cgi).
+
+### Git Tools
+
+To download the repository or a fork, you should have Git Tools installed. Download and install from [git-scm](https://git-scm.com/downloads).
+
+### Eclipse
+
+Optional: If you want or do use Eclipse.
+
+Download [Eclipse IDE for Enterprise Java Developers](https://www.eclipse.org/downloads/packages/release/2020-06/r/eclipse-ide-enterprise-java-developers) and install the following plugins:
```

Comment:
I was copying some boiler plate from the IBM FHIR team that had that section. For my development, I don't currently use any specialized plugins. M2E is probably the most important, but it isn't a strict requirement. There are plugins for PMD, CheckStyle, Jacoco (aka Ecclemma), but those aren't strict requirements. Where you thinking of anything else?

Reply:
I'd vote for ripping out this section. If we adopt certain plugins that end up being helpful then we can always add it back

Reply:
I personally don't see any need to impose plugins onto users.  In the end, the user decides what they want to use.  Our maven build will ensure all the necessary checks are being done.

I could maybe see an argument to mention something about PMD since we have a custom configuration file.

Reply:
Matt had the same feedback, so I just removed the blank plugins section.